src/lib/components/events/event-item.test.ts

```diff
@@ -0,0 +1,14 @@
+import { expect, test } from 'vitest';
+import { formatLocation } from './utils';
+
+test('can format bad locations', () => {
+  expect(formatLocation(null)).toBe('TBD');
+  expect(formatLocation('')).toBe('TBD');
+  expect(formatLocation(undefined)).toBe('TBD');
+  expect(formatLocation("   Your mom's house  ")).toBe("Your mom's house");
```

Comment:
How about "can format good locations"? and should this fall under that condition?

Reply:
Technically this isn't a good location, since it has white space on either end. Any data that has been parsed correctly should already be trimmed

Reply:
this is true, howabout a good location test case tho?

Reply:
New tests added

Reply:
lgtm : )